ref/Castle.Core-net45.cs

```diff
@@ -3756,24 +3756,24 @@ public abstract class InheritanceInvocation : Castle.DynamicProxy.AbstractInvoca
         public override object InvocationTarget { get; }
         public override System.Reflection.MethodInfo MethodInvocationTarget { get; }
         public override System.Type TargetType { get; }
-        protected virtual void InvokeMethodOnTarget() { }
+        protected abstract override void InvokeMethodOnTarget() { }
```

Comment:
Another unexpected change?

Reply:
Yes, but again, the change is towards the more correct declaration, [see the corresponding source code](https://github.com/castleproject/Core/blob/fae755cf2cf642068117f09711603a411ae10023/src/Castle.Core/DynamicProxy/Internal/InheritanceInvocation.cs#L50) which specifies `abstract override`, also.

Reply:
Makes sense since `InheritanceInvocation` is abstract.